hardware/cpu/intel/xeon_phi_3120a.pan

```diff
@@ -0,0 +1,10 @@
+structure template hardware/cpu/intel/xeon_phi_3120a;
+
+"manufacturer" = "Intel";
+"model" = "Intel(R) Xeon Phi Coprocessor 3120A @ 1.10 GHz";
+"speed" = 1100; # MHz
+"arch" = "x86_64";
+"cores" = 57;
+"max_threads" = 57;
+"type" = "Knights Corner"; # Intel codename
+"power" = 300; # TDP in watts
```

Comment:
Are you actually using these as system CPUs and managing an OS on them with Quattor?